storage/rocksdb/ha_rocksdb.cc

```diff
@@ -10260,8 +10260,9 @@ int ha_rocksdb::create_table(const std::string &table_name,
   DBUG_ENTER_FUNC();
 
   int err;
-  bool is_dd_tbl = dd::get_dictionary()->is_dd_table_name(
-      table_arg.s->db.str, table_arg.s->table_name.str);
+  bool is_dd_tbl = false;
+
+  if (ha_thd()->is_dd_system_thread()) is_dd_tbl = true;
```

Comment:
Consider using the ternary operator. Could you maybe also tell what made you do this change?

Reply:
I will exclude these line from this PR. Anyway, let me explain a little bit first, When DD upgrade occurs(either innodb->rocksdb or different version), we will 
1. create a temporarily db, 
2. create all of these DD table inside the db, 
3. "rename" temporarily db to mysql db after rename existing mysql db to another name.

These line change is "required" for 2nd step, otherwise, these DD table will created in default column family instead of __system__ column family. INNODB has similar issue when DD upgrade since it want to create these DD table in temporarily db under mysql tablespace..